OpenSim/Common/Component.h

```diff
@@ -1493,7 +1493,7 @@ OpenSim_DECLARE_ABSTRACT_OBJECT(Component, Object);
     void printSocketInfo() const;
 
     /** List all the inputs of this component and whether or not they are 
-    connected. Also list the connectee names for inputs that are connected. */
+    connected. Also list the (desired) connectee names for the intputs.      */
```

Comment:
Also, the user may interpret this to mean that there's a desired name for a particular connectee.